doc/UserGuide.md

```diff
@@ -57,14 +57,29 @@ Format: `list`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is case insensitive, the order of the keywords does not matter, only the name is searched,
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
 * `find John`<br>
-  Returns `John Doe` but not `john`
+  Returns `John Doe`
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find betsy tiM JoHn`<br>
+  Returns Any person having names `Betsy`, `Tim`, or `John`
+
+### Finding all persons that contains all specified tags: `findtag`
+Finds persons whose tags matches all the given tags.<br>
+Format: `find TAG [MORE_TAGS]`
+
+> The search is case sensitive, the order of the keywords does not matter, only the tags are searched,
```

Comment:
Any particular reason for this? You made the find case insensitive earlier; could have made this one also have the same behavior.

Reply:
Hi! Haha yes I could but I was thinking since tags are one-word, some users may combine a few words into their tags like #oneTwo, and I don't want to clash with the words that are meant to be #onetwo